### PR TITLE
feat(node): version 10.5.0 and 8.11.3

### PR DESCRIPTION
**Pull request by JacopoDaeli**

### CODE REVIEW

**Review by indexzero — APPROVED** [review]
Would like a 👍from @gergelyke before we merge.

**Review by jcrugzz — APPROVED** [review]
LGTM 👍

File: generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:10.3.0-alpine
+FROM node:10.5.0-alpine
```
Comment (silasbw) [inline generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile:1]: Are these generated files?  Did you update them manually or automatically?  The file path (generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile) doesn't match the version 10.5.0.

Similar feedback for the other Dockerfiles.

File: circle.yml [hunk 2]
```diff
@@ -7,10 +7,10 @@ machine:
 dependencies:
   override:
     - docker info
-    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/alpine/8.11.2-alpine && docker build -t godaddy/node:8.11.2-alpine-$version .;
-    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/alpine/10.3.0-alpine && docker build -t godaddy/node:10.3.0-alpine-$version .;
-    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/debian/8.11.2-debian && docker build -t godaddy/node:8.11.2-debian-$version .;
-    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/debian/10.3.0-debian && docker build -t godaddy/node:10.3.0-debian-$version .;
+    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/alpine/8.11.3-alpine && docker build -t godaddy/node:8.11.3-alpine-$version .;
+    - version=$(node -pe "($(cat package.json)).version"); cd generated-dockerfiles/alpine/10.5.0-alpine && docker build -t godaddy/node:10.5.0-alpine-$version .;
```
Comment (silasbw) [inline circle.yml:11]: Do these paths exist (see comment below)?

**Review by silasbw — CHANGES_REQUESTED** [review]

File: generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile (hunk 1 above)
Comment (JacopoDaeli) [reply generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile:1]: Right. I updated manually. Didn't realize there was a script to update these stuff for us. I run the script now and updated the PR. Should be all good now.
Should we add a `CONTRIBUTING.md` (or just a section to the README.md) for describing how to update Node docker images?

**Review by gergelyke — APPROVED** [review]

**Comment by gergelyke** [comment]
ty for the update Jacopo!

**Review by bmeck — APPROVED** [review]

**Review by silasbw — APPROVED** [review]
Yeah, updating the CONTRIBUTING.md in a separate PR.

**Comment by JacopoDaeli** [comment]
@silasbw 👍 ! I will send out a PR to add a `CONTRIBUTING.md` file.

**Pull request merged**